packages/beacon-node/src/sync/sync.ts

```diff
@@ -241,12 +232,12 @@ export class BeaconSync implements IBeaconSync {
       }
     }
 
-    // If we stopped being synced and falled significantly behind, stop gossip
+    // If we stopped being synced and fallen significantly behind, stop gossip
     else if (state !== SyncState.Synced) {
       const syncDiff = this.chain.clock.currentSlot - this.chain.forkChoice.getHead().slot;
       if (syncDiff > this.slotImportTolerance * 2) {
         if (this.network.isSubscribedToGossipCoreTopics()) {
-          this.logger.warn(`Node sync has fallen behind by ${syncDiff} slots`);
+          this.logger.warn(`Node sync has fallen behind by ${syncDiff} slots.`);
```

Comment:
It's not inconsistent, try `grep '\.");' packages/**/src/**/*.ts` you will find a lot more cases as well.

Reply:
```
> grep '\.");' packages/**/src/**/*.ts | wc -l
15
```

there are only 15 that's not a lot,  most are added by you it seems.

I don't think is very common to end logs with a period `.`, especially if those are not even full sentences like `"UnknownBlockSync enabled."`. It also doesn't really fit into the UX if we use a context object

e.g.
```
 this.logger.verbose("Added unknown block to pendingBlocks", {root: blockRootHex});
```
should this have a `.` (period)? I don't think it would read well
```
Added unknown block to pendingBlocks. root=0x1
```
We have other log separator like ` - `, I think a period only makes sense if you have 2 sentences in a single log message or error

Reply:
Yes there are few I don't know or remember if those are from me or not. 😄